internal/webhook/github/provider.go

```diff
@@ -79,3 +79,14 @@ func (g *Github) GetEvent(r *http.Request) (event.Event, error) {
 	}
 	return e, nil
 }
+
+func getNormalizedAction(action string) string {
```

Comment:
@corrieriluca I think we need, kind of the same mapping for Gitlab as there is also a reopen action

Reply:
This has already been done for GitLab, but the verbs used by GitLab are different

https://github.com/padok-team/burrito/blob/983ad07f65d31b27ef7772d27b1243d92f480130/internal/webhook/gitlab/provider.go#L77-L86